app/src/main/res/values/strings.xml

```diff
@@ -182,6 +182,30 @@
         To take photos, app needs write access to public storage</string>
     <string name="dialog__write_permission_for_camera__button_ok">OK</string>
 
+    <string name="dialog__alarm_permission_for_ping__title">
+        Permission required</string>
+    <string name="dialog__alarm_permission_for_ping__text">
+        Pinging machinery needs the permission to set exact alarms.
+        Alternatively, you can disable pinging in connection settings.</string>
+    <string name="dialog__alarm_permission_for_ping__positive_button">Grant permission</string>
+
+    <string name="dialog__notification_permission__title">Permission wanted</string>
```

Comment:
Demande d'autorisation

(_unclear why it's a different wording that the one for alarms_)

Reply:
It's not required, can continue without permission

Reply:
I see. The French wording is neutral enough to be used in both.